plugins/modules/route53.py

```diff
@@ -590,6 +590,9 @@ def main():
             command = 'UPSERT'
         else:
             command = command_in.upper()
+               
```

Comment:
```suggestion

```

Reply:
I think TTL does not required too.

```console
 def main():
     argument_spec = dict(
         state=dict(type='str', required=True, choices=['absent', 'create', 'delete', 'get', 'present'], aliases=['command']),
@@ -552,8 +541,6 @@ def main():
         'Weight': weight_in,
         'Region': region_in,
         'Failover': failover_in,
-        'TTL': ttl_in,
-        'ResourceRecords': [dict(Value=value) for value in value_in],
         'HealthCheckId': health_check_in,
     })

@@ -563,6 +550,9 @@ def main():
             DNSName=value_in[0],
             EvaluateTargetHealth=alias_evaluate_target_health_in
         )
+    else:
+        resource_record_set['TTL'] = ttl_in
+        resource_record_set['ResourceRecords'] = [dict(Value=value) for value in value_in]
```

Reply:
@Surgo Your patch fixes the issue #434 on my end.

Reply:
@boutetnico and @surgo thanks for the additional information.

It's extremely helpful when people who're actively using the modules contribute to testing bug fixes in a real environment.  While we can aim to make test suites as comprehensive as possible, they're almost never going to be perfect.